src/main/java/com/github/havardh/javaflow/phases/parser/java/ClassVisitor.java

```diff
@@ -102,7 +102,7 @@ private boolean isClass(ClassOrInterfaceDeclaration classOrInterfaceDeclaration)
   }
 
   private boolean isGetter(String name, String type) {
-    if (singletonList("boolean").contains(type) && name.startsWith("is")) {
+    if (asList("boolean", "Boolean").contains(type) && name.startsWith("is")) {
```

Comment:
could we introduce method name prefix of `has` too?

Reply:
I would not do it. It is not standard prefix according bean spec and it is not understood by jaxb.

Reply:
ok :)